platform/o.n.bootstrap/src/org/netbeans/NbClipboard.java

```diff
@@ -437,7 +461,8 @@ public void run() {
             catch (ThreadDeath ex) {
                 throw ex;
             }
-            catch (Throwable ignore) {
+            catch (InterruptedException | RuntimeException ex) {
+                log.log(Level.FINE, "systemClipboard not available", ex); // NOI18N
```

Comment:
Again, I'd do Level.INFO here, especially since we could here catch exceptions other than the known IllegalStateException from Clipboard.getContents.